templates/contact.html

```diff
@@ -0,0 +1,26 @@
+{% extends "_base.html" %}
```

Comment:
Personally i'd stick to the python quoting convention inside these python tags, even though it's in an html file. but take it or leave it.